src/Files.App/Extensions/MessageFormatExtensions.cs

```diff
@@ -25,11 +25,32 @@ public static class MessageFormatExtensions
 		/// </summary>
 		private static readonly CultureInfo _locale = new(AppLanguageHelper.PreferredLanguage.Code);
 
+		/// <summary>
+		/// Custom value formatters for the message formatter.
+		/// This class is used to customize the formatting of specific value types.
+		/// </summary>
+		private static readonly CustomValueFormatters _customFormatter = new()
+		{
+			// Custom formatting for number values.
+			Number = (CultureInfo _, object? value, string? style, out string? formatted) =>
+			{
+				if (style is not null && style == string.Empty)
+				{
+					// Format the number '{0, number}'
+					formatted = string.Format($"{{0:#,##0}}", value);
+					return true;
+				}
+				formatted = null;
```

Comment:
```suggestion

				formatted = null;
```

Reply:
I have to finally set up a regular formatting using a keyboard shortcut